common/component_interface_specs/package.xml

```diff
@@ -14,9 +14,10 @@
   <buildtool_depend>autoware_cmake</buildtool_depend>
 
   <depend>autoware_adapi_v1_msgs</depend>
+  <depend>tier4_control_msgs</depend>
 
   <test_depend>ament_lint_auto</test_depend>
-  <test_depend>autoware_lint_common</test_depend>
```

Comment:
This dependency is required for linting header files.

Reply:
Looks like a bad copy/paste from my side
Thanks!